packages/network-controller/tests/NetworkController.test.ts

```diff
@@ -2369,7 +2369,7 @@ describe('NetworkController', () => {
                 },
               });
 
-              expect(networkWillChange).toStrictEqual([[]]);
+              expect(networkWillChange).toBeFulfilled();
```

Comment:
We're awaiting the promise above, so this matcher isn't doing anything effectively (but make ESLint happy, so that there's some kind of assertion we're making). Thoughts on changing this to:

``` typescript
              const promiseForNetworkWillChange = waitForPublishedEvents({
                messenger,
                eventType: 'NetworkController:networkWillChange',
                operation: () => {
                  // Intentionally not awaited because we're capturing an event
                  // emitted partway through the operation
                  controller.rollbackToPreviousProvider();
                },
              });

              await expect(promiseForNetworkWillChange).toBeFulfilled();
```

Reply:
Good idea; done in https://github.com/MetaMask/core/pull/1347/commits/43bbcd66fc3c76460bc29ddef243b75bf08a814c